snuba/web/db_query.py

```diff
@@ -853,6 +853,7 @@ def _apply_allocation_policies_quota(
     Sets the resource quota in the query_settings object to the minimum of all available
     quota allowances from the given allocation policies.
     """
+    metrics.increment("db_query_single_query")
```

Comment:
Question: For query count is there another metric snuba.api.query.count?

Reply:
I can't find this metric. Do you mind pointing me to where you're looking at?

Reply:
https://github.com/getsentry/snuba/blob/63836d1d897d776c81617f6931b8cb27a738e958/snuba/querylog/__init__.py#L24-L69, Veneur [aggregates](https://github.com/stripe/veneur?tab=readme-ov-file#global-aggregation:~:text=Veneur%20can%20selectively%20aggregate%20global%20metrics%20to%20be%20cumulative%20across%20all%20instances%20that%20report%20to%20a%20central%20Veneur%2C%20allowing%20global%20percentile%20calculation%2C%20global%20counters%20or%20global%20sets.)